docs/quantum-consciousness.org

```diff
@@ -0,0 +1,197 @@
+
+Of course, all of this has tenuous value to me.  If we're being completely honest, I find that the notion of identifying with a quantum state should be terrifying more than anything else.  Entanglement networks, sub-universes in the Everett interpretation, all of these notions seem to be of problematic value.
+
+I think the main problem is a lack of critical examination of the intuitive notions.
+
+I started with the binding.  But, the fact that my qualia seem to suggest a close binding to my avatar does not mean, that it is the only place where my consciousness is bound.  Further, continuity /is/ tenuous.  Again, qualia seem to suggest that things like anæsthetics do in fact shut off consciousness, not just the recording in the meatsuit.  Dream-less sleep is yet another example.  So there's no need, no necessity for the intuitive notion to hold.  Additionally, for the strong binding to be true as well, one would have to observe the qualia for a /very/ long time.  Unfortunately, if this were not true, and one's conscsiousness indeed jumped to another avatar, your original avatar would continue acting within margin of error of what the physics would predict.  Even large personality changes are possible with time, and are not often considered a pathology: children grow up, so how do you /know/ that it is not /you/ -- your consciousness -- jumping into another avatar?
+
+Similarly, atomicity seems to be largely traditional.  Sure, /you/ might have the consciousness of a human, but that, in and of itself is a connected causal system with more than one instance.  To state that two identical copies of your body /cannot/ have a consciousness is preposterous, because you are a collection of small entities -- cells working in tandem, but not always.  Perhaps higher forms of organised matter have higher forms of consciousness, and lower forms too.  By that moniker, however, reduction to the absurd would suggest that something like an electron might have a consciousness[fn:16].  But there are no /good reasons/ to believe that /that/ is not the case.
+
+What is ifinitely more frustrating is that qualia seem to prevent us from doing any form of experiment to clarify if these notions are accidental or a pattern.  I can foresee an experiment in which I, construct a near-identical clone of myself, and transplant one neuron at a time into that clone.  From a qualia perspective I should be able to exactly pinpoint the moment when my concsiousness stopped, and /if/ it resumes, the moment that it did resume.  It is possible to iterate this experiment (in principle), to establish whether or not these notions are justified.  But the main problem is that the third principle is still in effect: I would come to the same conclusion as before, if I were a zombie.  The information that was obtained is itself a qualia, and is indistinguishable from a similar experiment with no consciousness involved!
```

Comment:
```suggestion
What is infinitely more frustrating is that qualia seem to prevent us from doing any form of experiment to clarify if these notions are accidental or a pattern.  I can foresee an experiment in which I, construct a near-identical clone of myself, and transplant one neuron at a time into that clone.  From a qualia perspective I should be able to exactly pinpoint the moment when my consciousness stopped, and /if/ it resumes, the moment that it did resume.  It is possible to iterate this experiment (in principle), to establish whether or not these notions are justified.  But the main problem is that the third principle is still in effect: I would come to the same conclusion as before, if I were a zombie.  The information that was obtained is itself a qualia, and is indistinguishable from a similar experiment with no consciousness involved!
```